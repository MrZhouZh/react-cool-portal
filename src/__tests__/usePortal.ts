import { renderHook } from '@testing-library/react-hooks';

import usePortal from '..';

describe('usePortal', () => {
  it('should...', () => {
    renderHook(() => usePortal);
  });
});
