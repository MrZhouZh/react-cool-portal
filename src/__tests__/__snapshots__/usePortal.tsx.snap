// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`usePortal should render portal correctly 1`] = `
<body>
  <div>
    <div
      class="ref"
    />
    <button
      data-testid="btn"
      type="button"
    >
      Show Ref
    </button>
    <div
      data-testid="is-show"
    />
  </div>
  <div
    id="react-cool-portal"
  >
    <div
      data-testid="target"
    >
      Test
    </div>
  </div>
</body>
`;
